integration_tests/testsuite/deploy_app.py

```diff
@@ -76,17 +86,62 @@ def _record_latency_to_bigquery(deploy_latency, language, is_xrt):
 def deploy_app_and_record_latency(appdir, language, is_xrt):
     start_time = time.time()
 
-    version = deploy_app(None, None, appdir, None)
+    version, url = deploy_app(None, None, appdir, None, 'GAE')
 
     # Latency is in seconds round up to 2 decimals
     deploy_latency = round(time.time() - start_time, 2)
 
     # Store the deploy latency data to bigquery
     _record_latency_to_bigquery(deploy_latency, language, is_xrt)
-    return version
+    return version, url
+
+
+def deploy_app_gae(yaml):
+    deployed_version = test_util.generate_version()
```

Comment:
Can you add a doc comment for the arguments.  E.g. 'yaml' is an absolute filename rather than a string containing yaml.